craft_archives/repo/package_repository.py

```diff
@@ -277,6 +312,30 @@ def _not_mixing_suites_and_path(
             raise _create_validation_error(url=values.get("url"), message=message)
         return suites
 
+    @root_validator
+    def _not_mixing_suites_and_series_pocket(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        suites = values.get("suites")
+        series = values.get("series")
+        pocket = values.get("pocket")
+        url = values.get("url")
+        if suites and (series or pocket):
+            raise _create_validation_error(
+                url=url, message="specify either suites or series and pocket."
```

Comment:
maybe we can make the message a bit clearer? Something like `suites cannot be used with series and pocket`